backend/src/modules/Announcements/service/AnnouncementValidationSchema.js

```diff
@@ -0,0 +1,11 @@
+import Joi from 'joi';
+
+export const announcementValidationSchema = Joi.object({
+  id: Joi.string(),
+  title: Joi.string().min(5).max(100).required(),
+  boardName: Joi.string().min(5).max(100).required(),
+  iconType: Joi.string(),
+  content: Joi.string().min(20).max(300).required(),
+  commentsIds: Joi.array().items(Joi.string()),
+  date: Joi.date(),
+});
```

Comment:
A to by miało więcej sensu (coś w tym stylu)?
Chodzi mi o zamysł przerobienia i nazwania funkcji, 
gdyż dzięku temu nie musimy wiedzieć co tak naprawdę się dzieje i jak ona działa.
Dzięki temu wywołujemy tylko funkcję a czy ona korzysta z Joi czy z regexa, czy z obu naraz to Nas już nie obchodzi).
Dzięki temu zaachowujemy zasadę "Dependency inversion principle" w SOLID i nie musimy sie martwić jak skorzystać z "announcementValidationSchema", i że jak wstawimy tam regexa (zamiast Joi), że nazwa schema nie bedzie pasować.

```suggestion
import Joi from 'joi';


export const validateAnnouncment = (props) =>{
 const announcementValidationSchema = Joi.object({
  id: Joi.string(),
  title: Joi.string().min(5).max(100).required(),
  boardName: Joi.string().min(5).max(100).required(),
  iconType: Joi.string(),
  content: Joi.string().min(20).max(300).required(),
  commentsIds: Joi.array().items(Joi.string()),
  date: Joi.date(),
  
 return announcementValidationSchema .validate()
});
```

Reply:
done